cmake/developer_package/compile_flags/sanitizer.cmake

```diff
@@ -5,20 +5,36 @@
 include(CheckCXXCompilerFlag)
 
 if (ENABLE_SANITIZER)
-    set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} -fsanitize=address")
-    check_cxx_compiler_flag("-fsanitize-recover=address" SANITIZE_RECOVER_ADDRESS_SUPPORTED)
-    if (SANITIZE_RECOVER_ADDRESS_SUPPORTED)
-        set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} -fsanitize-recover=address")
+    check_cxx_compiler_flag("/fsanitize=address" SANITIZE_ADDRESS_SUPPORTED)
+    if (SANITIZE_ADDRESS_SUPPORTED)
+        if (WIN32)
+            set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} /fsanitize=address")
+            check_cxx_compiler_flag("/fsanitize-recover=address" SANITIZE_RECOVER_ADDRESS_SUPPORTED)
+            if (SANITIZE_RECOVER_ADDRESS_SUPPORTED)
+                set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} /fsanitize-recover=address")
+            endif()
+            set(SANITIZER_LINKER_FLAGS "${SANITIZER_LINKER_FLAGS} /fsanitize=address")
```

Comment:
I don't think it is a linker flag for Visual Studio. To enable sanitizer for VS you only need a compiler flag.

Reply:
It's already placed in `if (NOT WIN32)`

Reply:
Fixed in last commit